apps/judicial-system/web/src/components/IndictmentCaseFilesList/IndictmentCaseFilesList.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useContext } from 'react'
+import { useIntl } from 'react-intl'
+
+import {
+  Case,
+  CaseFile,
+  CaseFileCategory,
+  completedCaseStates,
+  isCourtRole,
+} from '@island.is/judicial-system/types'
+import { Box, Text } from '@island.is/island-ui/core'
+
+import PdfButton from '../PdfButton/PdfButton'
+import { caseFiles } from '../../routes/Prosecutor/Indictments/CaseFiles/CaseFiles.strings'
```

Comment:
Consider moving all strings to `indictmentCaseFilesList.strings.ts`

Reply:
These strings are being used on the "Ákæra og fylgiskjöl" screen as well. Moving them means duplicating them.